### PR TITLE
Update fi.js

### PR DESCRIPTION
**Pull request by SirJypa**

### CODE REVIEW

**Comment by mty22** [comment]
Need native speaker to review

**Comment by mty22** [comment]
bonus kr up for grabs to review

**Comment by SuomiLammas25** [comment]
My ign is SheepWithAim

**Comment by pyrtsa-cell** [comment]
How can I help on this project?

**Comment by mty22** [comment]
merge conflicts resolved

File: fi.js [hunk 1]
```diff
@@ -1,472 +1,472 @@
...
+generic.logout.sessions Kirjaudu ulos kaikista istunnoista
+generic.username Syötä käyttäjätunnus
+generic.password Syötä salasana
+generic.register Rekisteröidy
+generic.login Kirjaudu Sisään
+generic.accuracy Tarkkuus
+player.waiting odotetaan pelaajia
+player.died Kuolit
+player.killed Sinut Tappoi
+player.killstreak {0} on a {1} Kill Streak
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: this one needs to remain in English

File: fi.js [hunk 2]
```diff
@@ -1,472 +1,472 @@
...
-settings.experimental.header Kokeellinen
-settings.experimental.aimFreezeFix Tähtäyksen Jähmettymisen Korjaus
-settings.experimental.mouseFlickFix Hiiren Napsauttamisen Korjaus
-settings.experimental.shaderRendering Instance Rendering
+settings.popup.header Ponnahdusikkunat
+settings.popup.freeKRPop Näytä Ilmaisen KR:n Ponnahdusikkuna
+settings.experimental.header Kokeellinen
+settings.experimental.aimFreezeFix Tähtäyksen Jähmettymisen Korjaus
+settings.experimental.mouseFlickFix Hiiren Napsauttamisen Korjaus
+settings.experimental.shaderRendering Instance Rendering
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: eng is fine here

File: fi.js [hunk 3]
```diff
@@ -1,472 +1,472 @@
...
 settings.experimental.staticRendering Static Rendering
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: if its not the best, then we'll leave it in eng

File: fi.js [hunk 4]
```diff
@@ -1,472 +1,472 @@
...
+settings.export Vie
+settings.none Asetuksia ei löydetty
+settings.backup Varmuuskopiointiasetukset
+settings.controls.change Näppäinasetukset
+settings.controls.press Paina mitä tahansa näppäintä
+settings.local.header Lokalisointi <span id="requiresRestart"><span style="color: #eb5656">*</span> vaatii uudelleenkäynnistyksen</span>
+settings.local.region Oletusalue
+settings.local.lang Kieli
+settings.network.header Verkko
+settings.network.sendRate Korkea Tickrate
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: ill revert to eng

File: fi.js [hunk 5]
```diff
@@ -1,472 +1,472 @@
...
+settings.mods.auto Automaattisesti Lataa Modit
+settings.mods.borders Scopen Reunat
+settings.mods.borderCol Scopen Reunat
+settings.mods.borderOpac Scopen Reunojen Läpinäkyvyys
+settings.mods.scope Scopen Kuva
+settings.mods.scope.paste Scopen Kuvan URL
+settings.mods.scope.opacity Scopen Läpinäkyvyys
+settings.mods.scope.width Scopen Kuvan Leveys
+settings.mods.scope.height Scopen Kuvan Korkeus
+settings.mods.dot Reticle Image
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: ok will leave it as is then for now

File: fi.js [hunk 6]
```diff
@@ -1,472 +1,472 @@
...
+settings.mods.timer Ajastinkuvakkeen Kuva
+settings.mods.timer.paste Ajastiskuvakkeen Kuvan URL
+settings.mods.usedamage Käytä Vahingon Overlayta
+settings.mods.damage Vahingon Overlayn Kuva
+settings.mods.damage.paste Vahingon Overlayn Kuvan URL
+settings.mods.overlay Pelin Overlayn Kuva
+settings.mods.overlay.paste Pelin Overlayn Kuvan URL
+settings.social.region Alueen Tilastot
+windows.settings.header Peliasetukset
+windows.mail.header Kruner-Posti
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: lets go with Krunker-Posti

File: fi.js [hunk 7]
```diff
@@ -1,472 +1,472 @@
...
+windows.controls.voice Puheääni
+windows.controls.list Pelaajalista
+windows.controls.board Tulostaulukko
+windows.controls.interact Vuorovaikutus
+windows.controls.interactSec Toissijainen Vuorovaikutus
+windows.controls.confirm Vahvista Vuorovaikutus
+windows.controls.drop Tiputa
+windows.controls.wepVis Vaihda Aseen Näkyvyyttä
 windows.controls.kickVoteY Vote Kick [Yes]
 windows.controls.kickVoteN Vote Kick [No]
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: once we can come up with something for `windows.controls.kickVoteY`, we can update `windows.controls.kickVoteN`

File: fi.js [hunk 8]
```diff
@@ -1,472 +1,472 @@
...
 windows.controls.kpdVoteY KPD Vote [Yes]
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: will leave it in eng for now since its for KPD

File: fi.js [hunk 9]
```diff
@@ -475,347 +475,347 @@ windows.controls.specFree Toggle Free Cam
...
+windows.store.premium Osta Premium
+windows.store.premium2 Krunker Premium
+windows.store.buyP Valitse näyttönimi ja paljon muuta ostamalla
+windows.store.buy Osta ja myy tuotteita ostamalla
+windows.store.wheels Palkintopyörät
+windows.store.unlock Avaa uusia aseiden skinejä ja esineitä
+windows.store.amount {0} KR:ää käyttäjällä
+windows.store.login ostaaksesi ja avataksesi tavaroita kauppapaikalta!
+windows.store.purchasing Ostetaan...
+windows.store.unboxed {0}{1} unboxattu
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: reverted to eng

File: fi.js [hunk 10]
```diff
@@ -475,347 +475,347 @@ windows.controls.specFree Toggle Free Cam
...
+windows.publish.mod.header Julkaise Modi
+windows.publish.mod.thumbnail Ei Pikkukuvaa
+windows.publish.mod.name Modin Nimi
+windows.publish.mod.paste Modin URL
+windows.publish.mod.update Julkaise/Päivitä
+windows.publish.mod.login luodaksesi ja ladataksesi modeja!
+windows.secondary.header Valitse Toissijainen
+windows.secondary.unlocked Avattu
 windows.secondary.customOnly Customs Only
 windows.secondary.premiumOnly Premium Only
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: will leave in ENG for now

File: fi.js [hunk 11]
```diff
@@ -475,347 +475,347 @@ windows.controls.specFree Toggle Free Cam
...
+server.config.noSecondary Poista Toissijaiset Käytöstä
+server.config.noStreaks Poista Tappoputket Käytöstä
+server.config.canSlide Sliding
+server.config.wallJP Seinähypyn Voimakkuus
+server.config.autoJump Automaattinen Hyppääminen
+server.config.bDrop Luotien Pudotus
+server.config.thirdPerson Kolmas Persoona
+server.config.requireCaptcha Vaadi Captcha
+server.config.nameTags Piilota Pelinimet
+server.config.kCams Kill Cams
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: leave in eng for now

File: fi.js [hunk 12]
```diff
@@ -475,347 +475,347 @@ windows.controls.specFree Toggle Free Cam
...
+server.config.scoreLimit Pisteraja
+server.config.keepTScore Pidä Tiimin Pisteet
+server.config.forceC Pakota Moden Hahmot
+server.config.logTim Lokiaika (Race, Parkour)
+server.config.hitBoxPad Hitbox Padding
+server.config.hpRegen Terveyden Regenerointi
+server.config.disableB Poista Reunat
+server.config.throwMel Heitettävät Meleet
+server.config.noCosm Disable Cosmetics
+server.config.tstCmp Test Mode
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: Kokeile Pelimuotoa

File: fi.js [hunk 13]
```diff
@@ -475,347 +475,347 @@ windows.controls.specFree Toggle Free Cam
...
+server.config.limitClasses Limit Classes
+server.config.tmSize Team Size
 server.config.fallDmg Fall Damage
 server.config.fallDmgThr Fall Damage Thresh
 server.config.noDraws No Draws
 server.config.bstOfR Best of
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: `Paras kolmesta` lets roll with this. Check it in-game once its implemented and review later please!

File: fi.js [hunk 14]
```diff
@@ -475,347 +475,347 @@ windows.controls.specFree Toggle Free Cam
...
-social.trades.accept Are you sure you want to accept this offer?
-stats.airdrops KR-Packages
-settings.quality.flapAnim Sniper Flap Animation
+social.login Kirjaudu/Rekisteröidy
+social.login.failed Kirjautuminen Epäonnistui
+social.login.buy Kirjaudu Sisään Ostaaksesi
+social.login.sell Kirjaudu Sisään Myydäksesi
+social.login.gift Kirjaudu Sisään Lahjoittaaksesi
+social.login.proccessing Kirjaudutaan sisään...
+social.since Since
```
[the remark this answers is not in the document]
Comment (mty22) [reply fi.js]: account creation date:

image

**Comment by mty22** [comment]
role sorted for SheepWithAim

**Comment by mty22** [comment]
kr sent

**Pull request merged**